libs/marinade-sdk/src/instructions/add_liquidity.rs

```diff
@@ -0,0 +1,38 @@
+use borsh::{BorshDeserialize, BorshSerialize};
+use solana_program::pubkey::Pubkey;
+
+#[derive(
+    InstructionData, Clone, Copy, Debug, Default, PartialEq, BorshSerialize, BorshDeserialize,
+)]
+#[discriminator([181, 157, 89, 67, 143, 182, 52, 72])]
+pub struct AddLiquidityData {
+    pub lamports: Option<u64>,
```

Comment:
It is not an option https://github.com/marinade-finance/marinade-anchor/blob/3b646ca387e5c56755d6482b171324a69948f469/programs/marinade-finance/src/lib.rs#L150

Reply:
+1, changed for the `lamports` field would be just `u64`